src/viam/robot/client.py

```diff
@@ -738,25 +757,29 @@ async def discover_components(
         queries: List[DiscoveryQuery],
     ) -> List[Discovery]:
         """
-        Get the list of discovered component configurations.
+        Get a list of discovered potential component configurations.
+        Only implemented for webcam cameras in builtin components. Returns module names for modules.
 
         ::
 
+            from viam.proto.robot import DiscoveryQuery
+
             # Define a new discovery query.
-            q = machine.DiscoveryQuery(subtype=acme.API, model="some model")
+            q = DiscoveryQuery(subtype="camera", model="webcam")
```

Comment:
good work

Reply:
For `subtype`, you could do `Camera.SUBTYPE.resource_subtype` which is a bit more verbose but fully typechecked.

This way also works, but just in case people don't know how to get the subtype? Up to y'all, just showing an additional way of doing it

Reply:
This way seems more straightforward but thank you